broadcasting/index.md

```diff
@@ -3,6 +3,7 @@ title: Membrane broadcasting tutorial
 description: >-
   Create your own media broadcasting solution!
 part: 8
+graphicPath: ../assets/images/Illo_fallback.png
```

Comment:
nope. Leave it empty here. A site should fallback to default image if a field is empty or absent.